bindings/python/test/test_arrow.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# from datetime import datetime, timedelta
+import unittest
+import unittest.mock as mock
+
+from pyarrow import int32, int64, schema as ArrowSchema, Table
+from pymongo import WriteConcern, DESCENDING
+import pymongo
+from pymongoarrow import aggregate_arrow_all, find_arrow_all, Schema
+
+from test import client_context
+from test.utils import WhiteListEventListener
+
+
+class TestExplicitArrowApi(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        if not client_context.connected:
+            raise unittest.SkipTest("cannot connect to MongoDB")
+        cls.cmd_listener = WhiteListEventListener('find', 'aggregate')
+        cls.getmore_listener = WhiteListEventListener('getMore')
+        cls.client = client_context.get_client(
+            event_listeners=[cls.getmore_listener, cls.cmd_listener])
+        cls.schema = Schema({'_id': int32(), 'data': int64()})
+        cls.coll = cls.client.pymongoarrow_test.get_collection(
+            'test', write_concern=WriteConcern(w='majority'))
```

Comment:
Why use `w=majority` explicitly?

Reply:
Force of habit. Any reason we shouldn't be? 
I thought it was good practice to use it to prevent some weird race condition where the first test doesn't see all the documents written during setup because they haven't propagated to a majority of nodes...

Reply:
That case only happens when you try to read your own writes from secondary and we added causal consistency to address that problem (w:majority doesn't fix this). The real reason to use w=majority is to avoid rollbacks; that is where the primary writes some data locally, responds ok:1, the client thinks the write is persisted, then before the write can propagate to a majority of nodes a new primary is elected and the write is later rolled back. I've never seen rollback happen in a driver test run ever so I don't think it's much of a concern but it is best practice. Feel free to keep it.

Reply:
👍🏽